packages/paypal-commerce-integration/src/paypal-commerce-script-loader.ts

```diff
@@ -109,7 +109,9 @@ export default class PayPalCommerceScriptLoader {
               )
             : availableAlternativePaymentMethods;
         const googlePayComponent: ComponentsScriptType = isGooglePayEnabled ? ['googlepay'] : [];
-
+        const cardFieldsComponent: ComponentsScriptType = initializesOnCheckoutPage
```

Comment:
Duplicate this code here `packages/google-pay-integration/src/google-pay-paypal-commerce/google-pay-paypal-commerce-script-loader.ts` please and add test if it is possible